tests/test_paperqa.py

```diff
@@ -1083,3 +1083,49 @@ def test_external_doc_index(stub_data_dir: Path) -> None:
     docs2 = Docs(texts_index=docs.texts_index)
     assert not docs2.docs
     assert docs2.get_evidence("What is the date of flag day?").contexts
+
+
+def test_context_inner_prompt(stub_data_dir: Path) -> None:
+
+    prompt_settings = Settings()
+
+    # try bogus prompt
+    with pytest.raises(ValueError, match="Context inner prompt can only"):
+        prompt_settings.prompts.context_inner = "A:"
+
+    # make sure prompt gets used
+    settings = Settings.from_name("fast")
+    settings.prompts.context_inner = "{name} @@@@@ {text}\nFrom: {citation}"
+    docs = Docs()
+    docs.add(stub_data_dir / "bates.txt", "WikiMedia Foundation, 2023, Accessed now")
+    response = docs.query("What country is Bates from?", settings=settings)
+    assert "@@@@@" in response.context
+    assert "WikiMedia Foundation, 2023" in response.context
+
+
+def test_evidence_detailed_citations_shim(stub_data_dir: Path) -> None:
+
+    # TODO: delete in v6
```

Comment:
Delete what, the entire test, or just a specific part of this test? Can you just update the comment to state one or the other

Reply:
Added comment - delete whole test